scripts/SREP/retry-pruning-cronjob/script.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+# Delete jobs that fail in openshift-sre-pruning namespace.
+
+set -x
+set -e
+set -o nounset
+set -o pipefail
+
+readonly PRUNING_NS=openshift-sre-pruning
+
+Builds_Pruner=()
+Deployements_Pruner=()
+FailedJobs=()
+
+Builds_Pruner+=($(oc get jobs -n "$PRUNING_NS" -o=jsonpath='{.items[?(@.metadata.ownerReferences[*].name=="builds-pruner")].metadata.name}'))
+Deployements_Pruner+=($(oc get jobs -n "$PRUNING_NS" -o=jsonpath='{.items[?(@.metadata.ownerReferences[*].name=="deployments-pruner")].metadata.name}'))
```

Comment:
This is not being used?

Reply:
The Deployements_Pruner one is not being used.  Just declared it like Builds_Pruner. Shall I remove it? Or echo it?